data/json/npcs/TALK_TRUE_FOODPERSON.json

```diff
@@ -353,7 +353,7 @@
         "trial": { "type": "PERSUADE", "difficulty": 0, "mod": [ [ "TRUST", 2 ], [ "VALUE", 2 ] ] },
         "success": {
           "topic": "TALK_FOODPERSON_SIDEKICK",
-          "effect": { "npc_add_var": "npc_succes_team_team", "type": "dialogue", "context": "foodperson", "value": "yes" }
+          "effect": { "npc_add_var": "npc_success_team_team", "type": "dialogue", "context": "foodperson", "value": "yes" }
```

Comment:
You didn't cause this bug, but I'm pretty sure it should be `npc_success_team` (without the extra `_team`).